NEXT_CHANGELOG.md

```diff
@@ -115,6 +115,12 @@ By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router
 
 By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/2096
 
+### Improve errors when subgraph returns non-2xx status code ([Issue #2117](https://github.com/apollographql/router/issues/2117))
```

Comment:
Can we convey this will only happen with non graphql subgraph responses ?

Reply:
Sure 👍